documents/operating.md

```diff
@@ -52,14 +52,202 @@ SECTION 4: 💵💵💵 MONEY 💵💵💵
 ---------------------------------------------------------------------
 SECTION 5: 🗣🗣🗣 Governing Process 🗣🗣🗣
 ---------------------------------------------------------------------
-- [ ] Types of decision
-- [ ] Records and documentation
-- [ ] Membership
-- [ ] Mechanisms for deliberation
-  - [ ] ... On GitHub
-  - [ ] ... On Slack
-- [ ] Use of emoji
-- [ ] Meetings
+- [x] Types of decision
+- [x] Records and documentation
+- [x] Mechanisms for deliberation
+  - [x] ... On GitHub
+  - [x] ... On Slack
+  - [x] ... Plotting Sessions
+- [x] Use of emoji
+
+### Deliberation Processes
+
+#### Slack
+Decisions made over slack are intended to be short term decisions that
+do not require intense deliberation or buy-in from the majority of the
+Hive Mind.
+
+1. A single pinned message is created, posing the entire question up
+for vote.
+
+2. Discussion happens entirely in a threaded response to the question.
+
+3. Voting is performed using emoji in response to a single pinned message
+that poses the entire content up for vote. Positive Emoji are used for a yes
+vote, Negative Emoji are used for a no vote, and Ambiguous Emoji are used
+for an abstention.
+
+4. The results of a vote are tallied and reported back inside of the thread
+in addition to being catalogued in `/votes/slack` as described later in this
+document.
+
+
+#### Plotting Session
+Plotting sessions are agenda driven and require a Quorum to occur.  They
+are led by the chair (as determined by the chain of command).
+
+The Hive Mind will use Robert's Rules of Order to run plotting sessions
+in an orderly and Robertly fashion.
+
+If the plotting session is held virtually, video should be enabled for all
+attendants where bandwidth allows.
+
+If the virtual platform enables costumes or static overlays on top of avatars,
+the meeting chairperson is to select an overlay to represent the Greek Chorus.
+This overlay should be selected to maximally hide the faces of the Greek Chorus.
+
+Each Corporate Overlord must wear overlays depicting wealth.  For example this
+could be a monacle, crown, or in the distant future a laser that fires precious
+gems.
```

Comment:
I'd rather this be phrased as an optional but strongly encouraged dress code, rather than a requirement for opulence.  Also that augmented reality overlays are a suitable alternative to actual items of attire.